lib/semantics/semantics.cc

```diff
@@ -12,16 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "semantics.h"
 #include "assignment.h"
 #include "canonicalize-do.h"
+#include "check-arithmeticif.h"
 #include "check-do-concurrent.h"
+#include "check-if-construct.h"
+#include "check-if-stmt.h"
 #include "expression.h"
 #include "mod-file.h"
 #include "resolve-labels.h"
 #include "resolve-names.h"
 #include "rewrite-parse-tree.h"
 #include "scope.h"
+#include "semantics.h"
```

Comment:
Wrong include order here too.

Reply:
Moved "include semantics.h" to the top of semantics.cc